src/shared/prosemirror-plugins/image-upload.ts

```diff
@@ -535,10 +535,10 @@ function createPlaceholder(): HTMLDivElement {
     const placeholder = document.createElement("div");
     placeholder.className = "ws-normal d-block m8";
     placeholder.innerHTML = `
-<div class="py6 px12 bg-black-050 bar-sm grid gsx gs8 d-inline-flex ai-center fw-normal fs-body1">
```

Comment:
This _may_ need `d-flex` since the `gs8` classes rely on that... Can we double check this?